Tools/CISupport/build-webkit-org/steps_unittest.py

```diff
@@ -958,6 +958,42 @@ def test_wpe_parameters(self):
         self.expectOutcome(result=SUCCESS, state_string='layout-tests')
         return self.runStep()
 
+    def test_default_timeout(self):
```

Comment:
nit: The other tests already cover the default timeout, probably can delete this and keep `test_site_isolation_timeout`.